index.html

```diff
@@ -41,19 +40,158 @@ <h1 id="idh1">
                 </a>
                 <ul class="icons">
                     <li><a href="https://twitter.com/carlosigreda" target="_blank" rel="noopener"><img
-                                src="/images/Twiter.svg" alt="Twiter"></a></li>
+                                src="./images/Twiter.svg" alt="Twiter"></a></li>
                     <li><a href="https://www.linkedin.com/in/carlos-jes%C3%BAs-igreda-rivas-323763116/" target="_blank"
-                            rel="noopener"><img src="/images/Linkedin.svg" alt="Linkedin"></a></li>
-                    <li><a href="https://medium.com/" target="_blank" rel="noopener"><img src="/images/Medium.svg"
+                            rel="noopener"><img src="./images/Linkedin.svg" alt="Linkedin"></a></li>
+                    <li><a href="https://medium.com/" target="_blank" rel="noopener"><img src="./images/Medium.svg"
                                 alt="Medium"></a>
                     </li>
                     <li><a href="https://github.com/CarlosIgreda" target="_blank" rel="noopener"><img
-                                src="/images/Github.svg" alt="Github"></a></li>
+                                src="./images/Github.svg" alt="Github"></a></li>
                     <li><a href="https://www.angellist.com/" target="_blank" rel="noopener"><img
-                                src="/images/AngelList.svg" alt="AngelList"></a></li>
+                                src="./images/AngelList.svg" alt="AngelList"></a></li>
                 </ul>
             </div>
         </section>
+        <section class="work s1">
+            <img src="./images/s1.svg" alt="tonic">
+            <h1>Tonic</h1>
+            <div class="info">
+                <h4>CANOPY</h4>
+                <ul class="extra">
+                    <li class="li1"><span>Back End Dev</span></li>
+                    <li class="li2"><span>2015</span></li>
+                </ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="skills">
+                <li>
+                    <a href="#"><button class="b1">html</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b2">css</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b3">javascript</button></a>
+                </li>
+            </ul>
+            <a href="#"><button class="b4">See Project</button></a>
+        </section>
+        <section class="work s2">
+            <img src="./images/s2.svg" alt="tonic">
+            <h1>Multi-Post <br> Stories</h1>
```

Comment:
- [ ] kindy use h2 tag here

Reply:
Done @tufoinnkuo10 . I've already changed it to an h2 tag.